👌 IMPROVE: Add default python repl continuation regex

s-weigand opened the pull request:
Just a little follow up to #88, adding the regex for line continuation of the python default repl (`r"\.\.\. "`) to the docs.

choldgraf commented:
any chance you'd be willing to document this pattern as well? I think it could be helpful for folks if we provided a few common patterns across Python, and the regexes that map onto them in the config. LMK what you think?

s-weigand commented on doc/index.rst:
```diff
@@ -157,7 +157,7 @@ For example this documentation uses the following configuration:
 .. code-block:: python
 
-   copybutton_prompt_text = r">>> |\$ |\[\d*\]: |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
+   copybutton_prompt_text = r">>> |\.\.\. |\$ |\[\d*\]: |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
```
I also added the pattern to the docs.

s-weigand commented:
Or do you mean something like a table explaining what matches what?
IMHO having a pattern in the docs which covers most use cases and users can simply copy-(button-)paste should be sufficient.
If people want to understand the regex in detail, they can use a website like [regex101](https://regex101.com/)  (which is also mentioned in the docs) and read the step by step explanation.

choldgraf commented:
A table would be awesome, but I just imagined a short paragraph to the effect of "this regex will match the following patterns" so users can know what to expect with it. Doesn't need to block this PR if you'd rather not do it though

s-weigand commented:
Ok can do ^^

choldgraf commented:
ah - that latest commit looks amazing 🌈 lemme know if this is ready to go

s-weigand commented:
I added a table with the prompt names, regex and match examples.
I also mentioned that users should look at regex101, for detailed explanation.
And I removed the 'old' style `\[\d*\]: ` matcher as per https://github.com/executablebooks/sphinx-copybutton/pull/88#discussion_r459137390

s-weigand commented:
Yap should be rdy to go ^^

choldgraf commented:
Amazing - thanks so much 👍
